scripts/gulp/tasks/interop.ts

```diff
@@ -9,10 +9,10 @@ var buffer = require('vinyl-buffer');
 var gts = require("gulp-typescript");
 import { join } from 'path';
 
-var tsProject = gts.createProject('tsconfig.json');
-
 import { buildConfig } from '../../build-config';
 
+var tsProject = gts.createProject(join(buildConfig.componentsDir, 'tsconfig.json'));
```

Comment:
I don't think it's going to matter here.

Reply:
see error while `dotnet build` of `Doc.sBuild` project:
![image](https://user-images.githubusercontent.com/1829209/173710995-06cc2e03-47b5-4b17-b2a2-6ffd19587184.png)